externals/global_params.php

```diff
@@ -82,6 +82,6 @@
      $URL_WIKI="https://wiki.gnucash.org/wiki/";
 
      // some defaults for metatags in case the page does not set them
-     $default_title= T_("Free Accounting Software");
-     $default_description= T_("<span class=\"gnucash\">GnuCash</span> is personal and small-business financial-accounting software, freely licensed under the <a href=\"https://www.gnu.org/\">GNU</a> GPL and available for GNU/Linux, BSD, Solaris, Mac OS X and Microsoft Windows.");
+     $default_title= T_("Copylefted Libre Software");
+     $default_description= T_("<span class=\"gnucash\">GnuCash</span> is double-entry accounting software for personal and small-business use. Freely licensed <a href=\"https://www.gnu.org/\">GPLv2+</a>. Available for GNU/Linux, BSD, macOS and Windows.");
```

Comment:
```suggestion
     $default_description= T_("<span class=\"gnucash\">GnuCash</span> is double-entry accounting software (licensed <a href=\"https://www.gnu.org/\">GPLv2+</a>) for people and small-businesses using Linux|GNU, BSD, macOS and Windows.");
```

Reply:
`GNU`: Either no family names or all family names (Apple, MS).

BTW Title is displayed in the browsers title bar, 
Title AND Description are used by google, facebook et al. as metadata for the first categorization

Reply:
Makes sense to rig that the other way around then. I think there are some SEO points to be had for repeating the title elsewhere and somesuch.